docs/user/tutorials/04-50-create-service-instance-with-custom-secret.md

```diff
@@ -0,0 +1,57 @@
+# Create a Service Instance with a Custom Secret
+
+To create a service instance, you must use the **btpAccessCredentialsSecret** field in the of the service instance spec. In it, you pass the Secret from the `kyma-system` namespace to create your service instance. You can use different Secrets for different service instances.
+
+> [!WARNING] 
+> Once you set a Secret name in the service instance, you cannot change it in the future.
+
+Adding the access credentials of the SAP Service Manager instance in your service instance results in displaying the subaccount ID to which the instance belongs in the status **subaccountID** field.
+
+> [!TIP]
+> For instructions on how to create a SAP BTP service operator Secret, see the [dedicated tutorial](04-20-create-btp-service-operator-secret.md).
+
+To create a service instance with a custom Secret, follow these steps:
+
+1. Get the access credentials of the SAP Service Manager instance with the `service-operator-access` plan from its service binding. Copy them as a JSON from the BTP cockpit. 
+
+2. Create the `creds.json` file in your working directory and save the credentials there. <!--is this step necessary for the managed offering; if not, what's the next step?-->
+
+3. When you have the Secret, create your ServiceInstance with the **btpAccessCredentialsSecret** field in the `spec` pointing to the newly created `my-secret` Secret and with other parameters as needed.
+
+   Here is an example of a ServiceInstance custom resource:
+
+   ```yaml
+   kubectl create -f - <<EOF
+   apiVersion: services.cloud.sap.com/v1
+   kind: ServiceInstance
+   metadata:
+     name: my-service-instance
+     namespace: default
+   spec:
+     serviceOfferingName: xsuaa
+     servicePlanName: application
+     btpAccessCredentialsSecret: my-secret
+   EOF
+   ```
+
+4. To verify that your service instance has been created successfully, run the following command:
+
+    ```bash
+    kubectl get serviceinstances.services.cloud.sap.com my-service-instance -o yaml
+    ```
+
+    You see the status `Created` and the message `ServiceInstance provisioned successfully`.
```

Comment:
To me, this seems to be the result of the whole procedure, not just of this step. This is the goal of the whole operation, isn't it?

Step 5 does NOT contribute to the goal of the operation (Create a Service Instance with a Custom Secret), right? I assume it undoes the steps before.